### PR TITLE
feat(dedicated-cloud): add region in HPC & some display fixes

### PR DESCRIPTION
**Pull request by carsso**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-15038
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Adding region data in HPC general information section and listing page.
Also fixes version display (previously not showing minor part of version number).
Refactoring listing page with translated columns and useful ones.

### General information view
 * Removed access policy section
 * Added region and location into location section
 * Fixed software solution section (showing minor part of version)

#### Before :
before-general-info

#### After :
after-general-info

### Listing page
 * New listing table with only useful columns
 * Added region and location columns
 * Added software solution column

#### Before :
before-global-list-censored
before-global-list-cols

#### After :
after-global-list-censored
after-global-list-cols
after-global-list-2-censored

### CODE REVIEW

**Review by dectotam — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html [hunk 1]
```diff
@@ -103,13 +103,35 @@
...
-    ></oui-tile-definition>
+    >
+        <oui-tile-description>
+            <dl class="mb-0">
+                <dt
+                    data-translate="ovhManagerPccDashboardGeneralInformation_location_zone"
+                ></dt>
+                <dd data-ng-bind=":: $ctrl.bindings.location"></dd>
+                <dt
+                    data-ng-if="!$ctrl.locationLoading && $ctrl.locationInfo"
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html:114]: If you use `loading` prop you don't need `ngIf` checks anymore
```suggestion
                    data-ng-if="!$ctrl.locationLoading && $ctrl.locationInfo"
```

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/list/dedicatedCloud-list.controller.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import get from 'lodash/get';
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/list/dedicatedCloud-list.controller.js:1]: if possible, try to use native JS functions
```suggestion
```

**Review by darsene — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html [hunk 3]
```diff
@@ -103,13 +103,35 @@
...
         data-term="{{:: 'ovhManagerPccDashboardGeneralInformation_location_term' | translate }}"
-        data-description="{{:: $ctrl.bindings.location }}"
-    ></oui-tile-definition>
-
-    <oui-tile-definition
-        data-term="{{:: 'ovhManagerPccDashboardGeneralInformation_accessPolicy_term' | translate }}"
-        data-description="{{:: $ctrl.bindings.accessPolicy }}"
-    ></oui-tile-definition>
+    >
+        <oui-tile-description>
```
[the remark this answers is not in the document]
Comment (carsso) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html:108]: Had to put it on <oui-tile>

**Review by darsene — APPROVED** [review]

**Review by dectotam — APPROVED** [review]

**Comment by carsso** [comment]
Capture d’écran 2024-08-29 à 11 50 57

Visual update following QA review

**Review by dectotam — DISMISSED** [review]

**Review by darsene — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Review by pauldkn — APPROVED** [review]

**Review by tibs245 — APPROVED** [review]

**Pull request merged**